<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgManageIcons</class>
 <widget class="QDialog" name="DlgManageIcons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manage node icons </string>
  </property>
  <widget class="QListWidget" name="listIcons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>211</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700000</width>
     <height>700000</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelQuickSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quick search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQuickSearch">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAcceptSelected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSetNothing">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>No Icon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>5</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelete">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Delete icon</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLoad">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Load from file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Save to file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
